### ISSUE TITLE
Cannot create hubert_pretrain_large() due to missing attribute LayerNorm.conv

### ISSUE THREAD

**Issue by arlofaria** [pytorch/audio#2885]
### 🐛 Describe the bug

Here's a minimal example:

```python
import torchaudio
torchaudio.models.hubert_pretrain_large()
```

The error is `AttributeError: 'LayerNorm' object has no attribute 'conv'`

### Versions

From what I can tell, this is a regression between versions 0.12.1 and 0.13.0

**Comment by nateanl** [comment]
oops, I made a mistake in `_init_hubert_pretrain_model` method. The line should be `if isinstance(module, components.ConvLayerBlock)` in 
https://github.com/pytorch/audio/blob/55e9978a68067f7f3b70ece667e25a12ed032a06/torchaudio/models/wav2vec2/model.py#L686

The error was not caught because there is no LayerNorm in the Base model, and I didn't test it on large and xlarge ones...

Let me make a fix and add tests for the pretrain models to make sure this won't happen again.

**Comment by nateanl** [comment]
addressed in #2886, could you try with the latest codebase? let me know if you meet any other issue

**Comment by arlofaria** [comment]
Seems to be working now, thanks!  I'll let you know if I run into any other issues.
